### ISSUE TITLE
possibilité d'ajouter plusieurs fois le tag du composant 

### ISSUE THREAD

**Issue by JaRoD41** [IVOOCodeHub/usv_migration_utilities#1]
pouvoir ajouter plusieur fois le tag du compo, quand on zone un composant, comme "bouton" ou "lien" par exemple, car actuellement le tag disparait de la liste et il n'est plus possible de l'ajouter à nouveau.

**Comment by Nilyss** [comment]
@JaRoD41 
Tu veux dire, pouvoir donner à une portion (ex: Header), 3x bouton en une sélection, plutôt que 3 sélections distinctes de bouton ?

**Comment by JaRoD41** [comment]
oui, dans l'idée. Un peu comme le projet avec les recettes de cuisine, il faudrait pouvoir ajouter plusieurs fois le même tag au lieu de le faire disparaitre de la liste. Ainsi, on aurait un "panier" avec par exemple Bouton, Bouton, Bouton.... etc... tu comprends ?
